### ISSUE TITLE
Winter is coming is resolved immediately and possible wildling icons need to be respected

### ISSUE THREAD

**Issue by gereon77** [Longwelwind/swords-and-ravens#206]
Errata v2 page 4:

![image](https://user-images.githubusercontent.com/22304202/74062635-fd5d0f80-49ee-11ea-9bbc-6fcef2300be6.png)

**Comment by gereon77** [comment]
@Longwelwind: From that point of view it doesn't make sense to fix the Winter is coming issue in #89. I would suggest implementing that flow which results in Winter is Coming will never be shown in Action panel anymore, just in Game Log. I remember you discussed that on Discord once. Was that the final decision? Can I close the bullet point in the UI issue?

**Comment by gereon77** [comment]
@Longwelwind 

Here is a possible solution https://github.com/gereon77/swords-and-ravens/tree/fix-winter-is-coming

As always I am open for coding improvements.

**Comment by gereon77** [comment]
Well in fact I don’t like that solution myself. I started it this way to not misuse the currentCardI counter but I guess it’s cleaner to use that and then the standard interface method and reset the counter after Winter is coming check and before next game state is triggered...

**Comment by gereon77** [comment]
I changed the implementation to use `currentCardI` and the standard WesterosCard method `execute`. I also tried to get rid of the while loop in `firstStart` and checked how it feels when I call `firstStart` again from `execute` but in the end fmpov the while loop was more readable and I kept that.

I tested the behaviour with a modified baseGameData with 10 Winter Is Coming cards and one other card in deck 1 and 2. As expected it took a while until the one other cards have been picked in both decks but then the game continued flowing. And also as expected there was a game log entry for each Winter is coming in event, so you will see the rare case that WiC is triggered twice or even more often.
